### PR TITLE
ANZ New Zealand import configuration

### PR DESCRIPTION
**Pull request by Dave4234**
Changes in this pull request:

1. Added 2 config files for ANZ New Zealand bank
2. Added readme file

@JC5

### CODE REVIEW

File: nz/anz/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# ANZ config files
+These config files provide configuration for exports from ANZ New Zealand accounts.
+
+## Which file?
+The file "default.json" is the standard config for most accounts (e.g. Go, Flexiloan).
```
Comment (JC5) [inline nz/anz/README.md:5]: If you want a new paragraph, be sure to add a double enter.

**Comment by JC5** [comment]
Nice work. Left one comment, if that's OK, I can merge it :+1:

File: nz/anz/README.md (hunk 1 above)
Comment (Dave4234) [reply nz/anz/README.md:5]: Cheers, have updated

**Comment by JC5** [comment]
@allcontributors add @Dave4234  for content

**Pull request merged**